ldi-core/ldes-client/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/filebased/FileManager.java

```diff
@@ -69,4 +76,25 @@ public void appendRecord(String file, String recordToAppend) {
 			createNewRecords(file, Stream.of(recordToAppend));
 		}
 	}
+
+	public void destroyState() {
+		try {
+			Path statePath = Path.of(stateDirectory).toAbsolutePath();
+			try (Stream<Path> list = Files.list(statePath)) {
```

Comment:
Can't we use "FileUtils.deleteDirectory" instead of looping and manually deleting the files?